test/assert.js

```diff
@@ -182,6 +183,26 @@ test('.throws()', function (t) {
 	t.end();
 });
 
+test('.throws() returns the thrown error', function (t) {
+	var expected = new Error();
+	var actual = assert.throws(function () {
+		throw expected;
+	});
+
+	assert.same(actual, expected);
```

Comment:
Oops wait. `assert.same()` means "deep equal". We want strict equality here. You should use `assert.is(actual, expected)` instead. Though the tests here are for AVA's `assert` module. Perhaps use `t.is(actual, expected)` which is from [`tap`](http://www.node-tap.org/asserts/)'s assertion library (AVA's tests are run using `tap`).

Same below.